src/Containers/docs/ReleaseNotes/v7.0.300.md

```diff
@@ -0,0 +1,6 @@
+# Microsoft.NET.Build.Containers 7.0.300
+
+This is the fifth public release of the .NET SDK containerization feature and the first release when the package is shipped as part of .NET SDK.
+Starting this release the package is released together with .NET SDK and has the matching version.
+
+This version brings the following new features and enhancements:
```

Comment:
@baronfel: anything to include here?

Reply:
Not yet, so I think it's ok to remove this line for now. If we are able to address any of the bugs during the approval window we can add them to this changelog at that time.